aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -629,6 +654,12 @@ impl VMChangeSet {
                             &mut group_entry.get_mut().inner_ops,
                             additional_inner_ops,
                         )?;
+
+                        let maybe_op_size = group_entry
```

Comment:
@vusirikala this is a fix

Reply:
@zekun000 "yeah this should be additional_maybe_group_op_size.unwrap()? since it can't be Deletion in this branch?"
I'm not sure, maybe it can be a deletion?